Forta-Bots-Challenge-3/src/network.ts

```diff
@@ -0,0 +1,81 @@
+import { providers, ethers, Contract, BigNumber } from "ethers";
+import { getEthersProvider } from "forta-agent";
+import {
+  ARBITRUM_DAI_ADDRESS,
+  L1_ARBITRUM_ESCROW_ADDRESS,
+  L1_OPTIMISM_ESCROW_ADDRESS,
+  OPTIMISM_DAI_ADDRESS,
+  ERC20_ABI,
+} from "./constants";
+
+export interface NetworkData {
+  chainId: number;
+  chainName: string;
+  daiContract: string;
+  escrowContract: string;
+  provider: any;
+}
+
+const OPTIMISM_DAI_DATA: NetworkData = {
+  chainId: 10,
+  chainName: "Optimism",
+  daiContract: OPTIMISM_DAI_ADDRESS,
+  escrowContract: L1_OPTIMISM_ESCROW_ADDRESS,
+  provider: new ethers.providers.JsonRpcProvider("https://mainnet.optimism.io"),
+};
+
+const ARBITRUM_DAI_DATA: NetworkData = {
+  chainId: 42161,
+  chainName: "Arbitrum",
+  daiContract: ARBITRUM_DAI_ADDRESS,
+  escrowContract: L1_ARBITRUM_ESCROW_ADDRESS,
+  provider: new ethers.providers.JsonRpcProvider("https://arb1.arbitrum.io/rpc"),
```

Comment:
Hey @msacco8, as per the challenge guidelines, it's important to avoid using a hardcoded endpoint. The bot should be able to run on all 3 chains independently (the chain is determined by the Forta node running the bot), with the _respective logic executed on each side_ (L1 - L2).

I recommend finding another way to communicate between the two sides and of course if you get stuck feel free to ping one of us